test/delabLinearIndependent.lean

```diff
@@ -4,9 +4,11 @@ set_option pp.unicode.fun true
 
 variable {K V : Type*} [DivisionRing K] [AddCommGroup V] [Module K V] {s : Set V} {x : V}
 
-variable (h : LinearIndependent K (fun b => b : s → V)) in
-/-- info: h : LinearIndependent K fun (b : ↑s) ↦ ↑b -/
-#guard_msgs in #check h
+-- FIXME temporarily removing this to get a build on nightly-testing
+
```

Comment:
@kmill is currently working on getting this fixed. It's on our list!

Reply:
What is the issue here?

Reply:
It doesn't work because the coercion delaborator was modified when upstreaming, and now interacts poorly with Kyle's projection notation delaborator. Kyle is working on this.

Reply:
Added link to Mathlib tracking issue.